etp-backend/src/main/sql/solita/etp/db/liite.sql

```diff
@@ -13,8 +13,8 @@ select distinct on (l.id) l.id, a.modifytime createtime,
 from liite l
 inner join audit.liite a on l.id = a.id
 inner join kayttaja k on a.modifiedby_id = k.id
-where l.energiatodistus_id = :energiatodistus-id
+where l.energiatodistus_id = :energiatodistus-id and l.deleted = false
```

Comment:
psql:ssä pitäisi toimia suoraan vain and not deleted - kannattaa kuitenkin kokeilla ensin

Reply:
Mahtaakohan olla standardia SQL:ää? Mielestäni paras pysyä standardissa niin paljon kuin mahdollista.